vuepress/docs/next/docs/getting-started/ent-diag.md

```diff
@@ -2,3 +2,26 @@
 sidebarDepth: 1
 ---
 
+# Diagnostics and Troubleshooting
+
+Diagnostic information is critical to analysis and troubleshooting, and the Entando CLI (ent) is a useful tool for debugging issues with an Entando instance. The [Getting Started](../../docs/getting-started/README.md) guide includes steps to install ent in a Multipass VM. The following commands can be run from this VM for insight into an Entando Application.
+
+| Command | Description
+| :- | :-
+| `ent app-info` | Display basic information about Kubernetes and Entando resources
+| `ent pod-info` | Display `kubectl describe` and `kubectl logs` for each Entando pod in a namespace
+| `ent diag` | Run diagnostics on the pods currently in an Entando namespace and prepare a diagnostic tar.gz
+| `ent pod grep --all "error\|fail"` | Locate error messages within EntandoApp pods 
+
+The command `ent diag` is a superset of operations to analyze, organize and output detailed pod information. In addition to calling `ent pod-info`, `ent diag` exports custom resources, ingresses, deployments, "previous" pod logs, namespace events, etc. It also generates diagnostics and returns log paths similar to the following:
```

Comment:
Not sure about the word superset>> it should be a set that is bigger than something else and I don't know that that's the case.

Reply:
it is a superset